src/components/discovery/DiscoveryQueryBuilder.js

```diff
@@ -130,6 +130,7 @@ class DiscoveryQueryBuilder extends Component {
             <Menu onClick={this.handleAddDataTypeQueryForm}>
                 {this.props.servicesInfo
                     .filter(s => (this.props.dataTypes[s.id]?.items ?? []).length)
+                    .filter(s => s.name !== "Bento Variant Service")
```

Comment:
Nitpicking. This loops over the array another time. Can be combined within the previous filter using `&&`

Reply:
I would add this new line filtering out Bento Variant Service won't be necessary after bentoV2 v2.6 release

Reply:
This is another readability vs clever one-liner issue. I usually opt for readability (and the array here is less than 10 items). Once the old variant service is removed this line can be removed entirely.